packages/docusaurus-theme-classic/src/theme/Footer/styles.module.css

```diff
@@ -13,3 +13,9 @@
 .footerLogoLink:hover {
   opacity: 1;
 }
+
+.footerLogoImg {
```

Comment:
Instead this, I would suggest within this PR, change `block` to `inline` in the stylesheet of `ThemedImage` component here https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-theme-classic/src/theme/ThemedImage/styles.module.css

Reply:
agree

would even use "initial" (but it's somehow the same)

Reply:
Would this need to be changed in both `classic` theme and `bootstrap` theme?

Reply:
the bootstrap theme is not too important right now, do not care about it ;)
I'll clean it up later when I'll be working on the theme gallery